index.js

```diff
@@ -282,11 +289,35 @@ function diffArgDescriptions(thisType, otherType, thisField, otherField, options
         return !!str;
     });
 }
+function diffArguments(thisType, otherType, thisField, otherField, options) {
+    const diffs = [];
+    if (!thisField.args || !otherField.args) {
+        return [];
+    }
+    let oldArgsMap = getArgumentMap(thisField.args);
+    let newArgsMap = getArgumentMap(otherField.args);
+    oldArgsMap.forEach(function(value, key) {
+        if (!newArgsMap.has(key)) {
+            const description = format('Argument `"{0} : {1}"` on type {2}.{3} has been removed from {4}.', value, key, thisType, thisField.name, options.labelForThis);
```

Comment:
we have to use language like "Argument missing from ..." instead of "Argument has been removed from" because not all consumers of this module will be comparing schemas for which that kind of language makes sense.